configs/defaults.yaml

```diff
@@ -28,5 +28,7 @@ headers:
     link: /education
   - name: Projects
     link: /projects
+  - name: Blog
+    link: https://medium.com/@datagoodatberkeley
```

Comment:
Oh man, this seems a bit dangerous since it's an external link. I'll modify the code a bit to fix this.